mpdsync.py

```diff
@@ -91,9 +94,9 @@ def main():
         sync(master, slaves)
         
     
-def get_settings(settings_file="settings.json"):
+def get_settings(settings_file=os.path.join(BaseDirectory.xdg_config_home, "mpdsync.json")):
```

Comment:
Since I prefer to use stdlib as much as possible, could you use [`os.path.expanduser()`](https://docs.python.org/3/library/os.path.html#os.path.expanduser) here instead?

Something like this:
```python
def get_settings(settings_file=os.path.expanduser('~/.config/mpdsync.json')):
```